File: NetBeans IDE/Evaluacion 2/EVA2_17_ENCONTRAR_NUMERO/Python/main.py
import random

random.seed()   #Prepare random number generator
iNumAlea = int(random.random() * 11)
for i in range(0, 2 + 1, 1):
    print("Adivina el numero")
    iNume = int(input())
    if iNume == iNumAlea:
        print("Felicidades, ganaste!!!")
        break
    if i == 2:
        print("Perdiste")

File: NetBeans IDE/Evaluacion 2/EVA2_17_ENCONTRAR_NUMERO/C++/main.cpp
#include <iostream>
#include <sstream>
#include <string>
#include <cstdlib>
#include <cmath>
#include <ctime>

using namespace std;

// Headers
string toString (double);
int toInt (string);
double toDouble (string);

int main() {
    srand(time(0));   //Seed random number generator"
    
    int iNumAlea, i;
    
    iNumAlea = rand() % 11;
    for (i = 0; i <= 2; i++) {
        cout << "Adivina el numero" << endl;
        int iNume;
        
        cin >> iNume;
        if (iNume == iNumAlea) {
            cout << "Felicidades, ganaste!!!" << endl;
	    break;
        }
        if (i == 2) {
            cout << "Perdiste" << endl;
        }
    }
    return 0;
}

// The following implements type conversion functions.
string toString (double value) { //int also
    stringstream temp;
    temp << value;
    return temp.str();
}

int toInt (string text) {
    return atoi(text.c_str());
}

double toDouble (string text) {
    return atof(text.c_str());
}
